sdk/arm/network/2020-03-01/armnetwork/example_firewallpolicies_test.go

```diff
@@ -103,7 +103,7 @@ func ExampleFirewallPolicyRuleGroupsOperations_BeginDelete() {
 
 func createFirewallPolicy(rgName, loc, policyName string) *FirewallPolicy {
 	fpClient := getFirewallPoliciesOperations()
-	fpResp, err := fpClient.BeginCreateOrUpdate(
+	poller, err := fpClient.BeginCreateOrUpdate(
```

Comment:
Isn't it a bit confusing to name this poller? It's more like pollerResp since this isn't the poller itself. Just for clarity in the example.

Reply:
Fixed.